cli/bazelisk/bazelisk.go

```diff
@@ -62,6 +64,39 @@ func Run(args []string, opts *RunOpts) (exitCode int, err error) {
 	return core.RunBazelisk(args, repos)
 }
 
+func BazelInfo(requestInfos []string) (map[string]string, error) {
+	bazelArgs := append([]string{"info"}, requestInfos...)
+	stdout := &bytes.Buffer{}
+	stderr := &bytes.Buffer{}
+	opts := &RunOpts{
+		Stdout: stdout,
+		Stderr: stderr,
+	}
+	// Prevent Bazelisk `log.Printf` call to write directly to stderr
+	oldWriter := log.Writer()
+	log.SetOutput(stderr)
+	defer log.SetOutput(oldWriter)
+
```

Comment:
it seems OK to me to put this logic in `Run()` since to me the expected behavior of `RunOpts` would be that all stderr/stdout from bazelisk, including go logs, goes to the stdout/stderr writers passed in. wdyt?

Reply:
hmm I tried fitting it in if RunOpts.stderr is set 🤔